sunpy/instr/aia/read_genx.py

```diff
@@ -0,0 +1,386 @@
+from __future__ import (absolute_import, division, print_function,
```

Comment:
Ah ok, fair point. I can see where this name might be a bit confusing.

We discussed trying to come up with a general tool for reading genx files and putting it somewhere else, maybe `sunpy/util`, but I think this would require a bit of time. The main problem is still that these genx files cannot be read "as is" and need to be reformatted as IDL save files.